GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -375,17 +369,11 @@ VirtualFramebuffer *FramebufferManagerCommon::DoSetRenderFrameBuffer(const Frame
 		// Lookup in the depth tracking to find which VFB has the latest version of this Z buffer.
 		// Then bind it in color-to-depth mode.
 		//
-		// We are going to do this by having a special render mode where we take color and move to
+		// We do this by having a special render mode where we take color and move to
 		// depth in the fragment shader, and set color writes to off.
 		//
-		// We'll need a special fragment shader flag to convert color to depth.
-
-		for (auto &depth : this->trackedDepthBuffers_) {
-			if (depth->z_address == params.fb_address && depth->z_stride == params.fb_stride) {
-				// Found the matching depth buffer. Use this vfb.
-				vfb = depth->vfb;
-			}
-		}
+		// We use a special fragment shader flag to convert color to depth.
+		vfb = GetLatestDepthBufferAt(params.fb_address /* !!! */, params.fb_stride);
```

Comment:
nit: This looks as much like "important, fb was intentional here" as "TODO, gotta change something here."

-[Unknown]